code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1331,7 +1331,7 @@
 
 /datum/reagent/freon
 	name = "Freon"
-	description = "A powerful heat adsorbant."
+	description = "A powerful heat absorbent."
```

Comment:
adsorbent is the right one (yes I did a typo in the name but the d is right)

Reply:
I don't believe adsorbent is correct. From https://www.merriam-webster.com/dictionary/adsorption (Adsorbent just points to adsorb which then points to adsorption)
"the adhesion in an extremely thin layer of molecules (as of gases, solutes, or liquids) to the surfaces of solid bodies or liquids with which they are in contact "

Reply:
Mmm you could be right, I was thinking of Freon molecules to distribute the heat "around" them by electrons and atoms movements... But yeah the definition has it different, nevermind then replace d with b